documentation-site/examples/spinner/next-span.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+
+import * as React from 'react';
+import {StyledSpinnerNext} from 'baseui/spinner';
+
+export default () => (
+  <p>
+    <StyledSpinnerNext $as="span" $style={{display: 'block'}} />
```

Comment:
I'd incline not to use `$style` in the examples. Can we replace it with `withStyle`?

Reply:
just out of curiosity, what's wrong with `$style`? we document it on the styletron website (https://www.styletron.org/react/#style-prop), so I think one would assume it's part of the public API?

Reply:
I know that we document it in Styletron docs, but it's not a part of the base web API we've been encouraging people to use. I also think it may create more confusion around how and when one would use it. It's more of an internal implementation imho.